packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -196,7 +196,7 @@ const DesignCard: React.FC< DesignCardProps > = ( {
 	const isDefaultVariation = isDefaultGlobalStylesVariationSlug( selectedStyleVariation?.slug );
 
 	const isLocked = isLockedStyleVariation( {
-		isPremiumTheme: design.is_premium,
+		isPremiumTheme: design?.design_tier === 'premium',
```

Comment:
this explicitly checks for premium and not the starter tier because it governs how global styles is displayed - you still need the premium plan to use gs on a free/starter tiered theme.